templates/security/fips.html

```diff
@@ -73,33 +98,53 @@ <h2>Access FIPS images on the cloud</h2>
       <p>Ubuntu Pro FIPS 20.04</p>
       <hr>
       <div style="padding-top: 1rem;">
-        <a class="p-link--external" style="padding-top: 1rem;" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List">Under validation</a>
+        <a class="p-link--external" style="padding-top: 1rem;" href="https://csrc.nist.gov/Projects/cryptographic-module-validation-program/modules-in-process/Modules-In-Process-List">In progress</a>
       </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip">
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-6">
+      <h2>FIPS certification and CIS compliance with Ubuntu</h2>
+      <p>Learn about Ubuntu CIS and FIPS certified components to enable operating under compliance regimes like FedRAMP, HIPAA, PCI and ISO. Get all of your compliance questions answered in our upcoming webinar to ensure you and your team are, and remain, compliant.</p>
+      <a class="p-button--positive js-invoke-modal" href="/security/certifications#get-in-touch">Contact us</a>
+    </div>
+    <div class="col-6 u-hide--small u-align--center">
+      <div class="jsBrightTALKEmbedWrapper" style="width:100%; height:100%; position:relative;background: #ffffff;">
+        <script class="jsBrightTALKEmbedConfig" type="application/json">
+          { "channelId" : 6793, "language": "en-US", "commId" : 432536, "displayMode" : "standalone", "height" : "250" }
+        </script>
+        <script src="https://www.brighttalk.com/clients/js/player-embed/player-embed.js" class="jsBrightTALKEmbed"></script>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
```

Comment:
I feel this should be a `p-strip` to keep the sections separate and then alternate the following ones